CONTRIBUTING.md

```diff
@@ -331,7 +331,7 @@ even patch releases may contain [non-backwards-compatible changes](https://semve
 ### Creating a new release
 
 1. Install `uv`: `curl -LsSf https://astral.sh/uv/install.sh | sh`
-1. Run `./scripts/release/bump.sh`; this command will:
+1. Run `./scripts/release.sh`; this command will:
     - Generate a temporary virtual environment with `rooster`
```

Comment:
```suggestion
```
it is cached in user's data dir now.

Reply:
It's technically not even cached, idk if this matters though.